bitcoin/CHANGELOG.md

```diff
@@ -1,7 +1,102 @@
-# unreleased
-
-- Bump MSRV to Rust 1.56.1
-- Remove "no-std" feature
+# 0.32.0 - TODO: Add date when we do final release
+
+- Bump MSRV to Rust 1.56.1 [#2188](https://github.com/rust-bitcoin/rust-bitcoin/pull/2188)
+- Remove "no-std" feature [#2233](https://github.com/rust-bitcoin/rust-bitcoin/pull/2233)
+- Enforce displaying `Amount` with trailing zeros [#2604](https://github.com/rust-bitcoin/rust-bitcoin/pull/2604)
+
+## Things you will almost certainly hit when upgrading
+
+- Rename `Signature` field names [#2338](https://github.com/rust-bitcoin/rust-bitcoin/pull/2338)
+  - `sig` becomes `signature`
+  - `hash_ty` becomes `sighash_type`
+- Rename `txid` to `compute_txid` [#2366](https://github.com/rust-bitcoin/rust-bitcoin/pull/2366)
+- In hardcoded BIP-32 derivation paths: Remove `m/` prefix requirement [#2451](https://github.com/rust-bitcoin/rust-bitcoin/pull/2451)
```

Comment:
In 0073a17e2066b405aee59d0bb6012260b85c696e:

I don't know that anybody will hit this. It's certainly not a "you will almost certainly hit" thing. Somebody would have filed an issue much sooner about the m-prefix if it was.

Reply:
Oh, I can move it. I hit it the miniscript upgrade and it was surprising.

Reply:
I'll use this as the first PR verifying I have the new branching strategy internalised correctly (I've explained it again below).